copier/vcs.py

```diff
@@ -10,14 +10,33 @@
 from packaging import version
 from packaging.version import InvalidVersion, Version
 from plumbum import TF, ProcessExecutionError, colors, local
-from plumbum.cmd import git
 
 from .errors import DirtyLocalWarning, ShallowCloneWarning
 from .types import OptBool, OptStr, StrOrPath
 
+
+def get_git():
+    """Gets `git` command, or fails if it's not available"""
+    try:
+        from plumbum.cmd import git
+        return git
+    except ImportError as exc:
+        raise OSError("git is not available") from exc
+
+try:
+    GIT_VERSION = Version(re.findall(r"\d+\.\d+\.\d+", get_git()("version"))[0])
+except OSError:
+    import warnings
+
+    warnings.warn(
+        "git command not found, some functionality might not work", UserWarning
+    )
+
+    GIT_VERSION = None  # type: ignore
```

Comment:
I think this is better done in a function. `def git_version(): ...`

This way it's lazy and you actually get an exception if you really need a git version for whatever reason.